spec/system/kudo_create_spec.rb

```diff
@@ -2,14 +2,17 @@
 
 RSpec.describe 'Kudo management', type: :system do
   before do
-    employee = create(:employee)
-    login_as(employee)
     driven_by(:selenium_chrome_headless)
   end
 
   it 'enables me to create kudos' do
+    employee = create(:employee)
+    visit '/employees/sign_in'
+    fill_in 'Email', with: employee.email
+    fill_in 'Password', with: 'password'
+    click_button 'Log in'
```

Comment:
to samo tutaj, znów testujemy logowanie się w miejscu gdzie nie musimy go testować, przez co wydłużamy testy, test powinen testować tylko jedną rzecz